app/donate/steps/Step1PersonalInfo.tsx

```
@@ -236,22 +301,23 @@ export default function Step1PersonalInfo() {
                 });
                 if (errors.zipCode) setErrors({ ...errors, zipCode: "" });
               }}
             />
-            {errors.zipCode && <p className="text-red-500 text-sm mt-1">{errors.zipCode}</p>}
+            {errors.zipCode && <p className="form-error text-red-500 text-sm mt-1">{errors.zipCode}</p>}
           </div>
         </div>
 
         <p className="text-sm italic text-gray-700 mt-4">
-          We have listed only the towns we pick up from. If your town is not listed, please contact volunteer@journeyhomect.org to see if a pick up is possible
+          Only towns/cities within our pickup range are listed as an option above. If your town is not listed, you can select it through the Other option. If your town isn't listed, but you need a pickup, please contact volunteer@journeyhomect.org to see if a pickup is possible.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)